### PR TITLE
feat: add account generation script

### PR DESCRIPTION
**Pull request by krtk6160**
This script has already been run on a live environment and works as expected

### CODE REVIEW

File: src/debug/create-wallets.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  if (args.length === 5) {
+    const numWallets = parseInt(args[2])
+
+    const disbursementAmount = checkedToSats(parseInt(args[4]))
+    if (disbursementAmount instanceof Error) return disbursementAmount
+
+    const disburserWalletId = checkedToWalletId(args[3])
+    if (disburserWalletId instanceof Error) return disburserWalletId
+
+    const wallets = await generateWallets(numWallets)
```
Comment (nicolasburtey) [inline src/debug/create-wallets.ts:95]: probably need to dump this in a csv

File: src/debug/create-wallets.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    const account = await User.create({ phone })
+
+    const btcWallet = await WalletsRepository().persistNew({
+      accountId: account._id,
+      type: WalletType.Checking,
+      currency: WalletCurrency.Btc,
+    })
+    if (btcWallet instanceof Error) return btcWallet
+
+    const usdWallet = await WalletsRepository().persistNew({
```
Comment (nicolasburtey) [inline src/debug/create-wallets.ts:59]: maybe usd wallet the default wallet

File: src/debug/create-wallets.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    console.log("verify", jwt.verify(jwtToken, JWT_SECRET))
+  }
+
+  return wallets
+}
+
+const main = async () => {
+  const args = process.argv
+  if (args.length === 5) {
```
Comment (nicolasburtey) [inline src/debug/create-wallets.ts:86]: can we also have an arg to set the default level of the account when they are generated?

File: src/debug/create-wallets.ts (hunk 2 above)
Comment (krtk6160) [reply src/debug/create-wallets.ts:57]: Done

File: src/debug/create-wallets.ts (hunk 3 above)
Comment (krtk6160) [reply src/debug/create-wallets.ts]: done

File: src/debug/create-wallets.ts (hunk 1 above)
Comment (krtk6160) [reply src/debug/create-wallets.ts]: done

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**